### ISSUE TITLE
eparties - cleanup and consolidation of Smithsonian records

### ISSUE THREAD

**Issue by rondlg** [fieldmuseum/EMu-Documentation#918]
### Discussed in https://github.com/fieldmuseum/EMu-Documentation/discussions/911

<div type='discussions-op-text'>

<sup>Originally posted by **cmaves3** October  6, 2023</sup>
Description: There are many duplicate records for the Smithsonian Institute across several departmental master lists. We should work to consolidate these records. Some matches are easy to identify while some may require a bit of looking into and/or consulting across departments. See link to spreadsheet. The records have been sorted into preliminary groups but you are not restricted to merging within those groups.

[See Consolidating Parties Sheet > Organisation - Smithsonian tab](https://docs.google.com/spreadsheets/d/11pFwe-9bqg7iO-BqqlV6_tSI7N7Ma6X51zptJGDBvJM/edit#gid=1645755592)

In conjunction with this, we must decide how to structure these records in the Organisation tab. SI is an entity that has many constituent parts such as the National Museum of Natural History (NMNH). The museum has departments including Paleobiology. If you structure the EMu record with SI, NMNH, and Paleo as three separate fields then “Paleobiology” ends up in the Branch field so searching for it in the Department field will not yield results. If we want the dept name to be in the dept field then we will need to uniformly concatenate SI and NMNH (Ex. Smithsonian Institution - National Museum of Natural History).</div>

**Comment by seanmkeogh** [comment]
Invert IRNs 158034, 205449, 4007, 150581, 150647 merged with suggested IRNs and deleted

**Comment by rbaquiran** [comment]
Again, Insects and Insect used FMNH Masters are done. 
Thanks for the address clarification and MCR tip, Chris. Seeing as MRCs are unique to each individual, not the department, they are now reflected in their associated addresses.

**Comment by rondlg** [comment]
@cmaves3 Checking in to see if this data cleanup task has be completed?

**Comment by cmaves3** [comment]
> @cmaves3 Checking in to see if this data cleanup task has be completed?

No. Made good progress, though. Need to regroup and get the rest finished with more direct instruction.

**Comment by rondlg** [comment]
@cmaves3 Any chance this is complete?

**Comment by cmaves3** [comment]
@rondlg Unfortunately, this has been stuck on the back-burner. I've cleaned up the sheet and can now continue to work on the remaining records.
